src/events/guildMemberRemove.js

```diff
@@ -0,0 +1,40 @@
+const { Events, EmbedBuilder } = require("discord.js");
+const moment = require("moment");
+
+/*
+	Event: guildMemberRemove
+	Uses Database?: No
+	Description: Emitted whenever a member leaves a guild, or is kicked.
+*/
+
+module.exports = {
+  name: Events.GuildMemberRemove,
+
+  execute(member) {
+    // Ignore bots.
+    if (member.user.bot) {
+      return;
+    }
+
+    // getting channel info
+    const channel = member.guild.channels.cache.get(
+      process.env.GENERAL_CHANNEL_ID
```

Comment:
Set to this the BOT Channel ID. 
It can be pretty demoralizing seeing people leave the server in main, so we'll just add it the bots channel.

Reply:
Corrected.